src/ReactNativeZoomableView.tsx

```diff
@@ -349,11 +349,28 @@ class ReactNativeZoomableView extends Component<ReactNativeZoomableViewProps, Re
         const currentElementWidth = originalWidth * changeObj.zoomLevel;
         const currentElementHeight = originalHeight * changeObj.zoomLevel;
 
+        // gentle reminder for the following logic:
+        //    RATIO = W / H
+        //    H = W / RATIO
+        //    W = H * RATIO
+
+        const aspectRatioOnMount = originalHeight && originalWidth ? originalWidth / originalHeight : 1;
+        const { contentAspectRatio = aspectRatioOnMount } = this.props;
```

Comment:
Could you please put this to the top of the method.
I would like to follow the best practice to have all external (prop) definitions at the top of the methods to know what we really use within our methods immediately.
(we need to convert some of the other occurences this.props... to, but let's start here.